core/stream/pipeline.pl

```diff
@@ -184,13 +184,23 @@ (&@)
 #   lzo:   89 4c 5a 4f
 #   lz4:   04 22 4d 18
 #   xz:    fd 37 7a 58 5a
-#   xlsx:  50 4b 03 04 14 00 06 00 (actually this is all MS OOXML)
```

Comment:
I removed the last two bytes because gnumeric had exported xlsx with `50 4b 03 04 14 00 00 00` instead of the `06`. Six bytes of magic is pretty good (and `PK\03\04` is the PKzip archive header, so we'll be able to short-circuit that one earlier if we end up handling zip archives).